nextmv/nextmv/options.py

```diff
@@ -624,10 +627,70 @@ def merge(self, new: "Options") -> "Options":
 
         self.options += new.options
 
-        self._parse()
+        if not skip_parse:
+            self._parse()
 
         return self
 
+    @classmethod
+    def merge_all(cls, *options: "Options", skip_parse: bool = False) -> "Options":
```

Comment:
I think that we should change this to go with one of the following:
1. Make this a proper method on the class and merge all incoming options to the current class. Like "merge", but receiving more than one options object. 
2. Make it a standalone function outside of the class. 

The reason is that it being a classmethod I think is unnecessary. Id it were up to me, I would go with 1 for consistency and maybe change the name to "merge_multiple".

Reply:
Adding it to the class / object feels a bit weird, as there is already a way to merge options. Let me check whether there is a non-breaking way to extend it though.

Regarding the static method vs. standalone: I don't have a strong opinion. However, we are already offering other methods that are similarly tied to the class definition as static methods on the class, so, I actually feel that this is more in line with the style that was used so far.

I'll try to remove the method though and merge it with the other one.